src/current/_includes/releases/v23.1/v23.1.0-beta.2.md

```diff
@@ -30,7 +30,7 @@ Release Date: April 17, 2023
 
 <h3 id="v23-1-0-beta-2-performance-improvements">Performance improvements</h3>
 
-- Pubsub sink [changefeeds](https://www.cockroachlabs.com/docs/v23.1/create-changefeed) can now support higher throughputs by enabling the `changefeed.new_pubsub_sink_enabled` [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings). [#100930][#100930]
+- Google Cloud Pub/Sub sink [changefeeds](https://www.cockroachlabs.com/docs/v23.1/create-changefeed) can now support higher throughputs by enabling the `changefeed.new_pubsub_sink_enabled` [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings). Enabling this setting will cause changefeeds to use the v2 Pub/Sub sink, which has capitalized top-level fields in the message: `{Key: ..., Value: ..., Topic: ...}`. The top-level message fields in v1 Pub/Sub are all lowercase: `{key: ..., value: ..., topic: ...}`. You may need to reconfigure downstream systems to parse the new message format. [#100930][#100930]
```

Comment:
I'd prefer not to refer to "v1" and "v2" externally, since although we have separate implementations for them internally, externally it seems like it should be seen as pubsub with some improvements/different behavior. Maybe refer to them in relation to the cluster setting being enabled/disabled? What do you think?

Reply:
Got it, I think I have swapped out all those instances for alternative language.

Reply:
Thanks, sounds better to me.